reana-cwl.yaml

```diff
@@ -1,14 +1,7 @@
 version: 0.2.0
-code:
-  files:
-   - code/gendata.C
-   - code/fitdata.C
 inputs:
   parameters:
     input: workflow/cwl/input.yml
-outputs:
-  files:
-   - outputs/plot.png
 environments:
```

Comment:
Shall we remove environment here as well? It is already present in the workflow spec https://github.com/reanahub/reana-demo-root6-roofit/blob/master/workflow/cwl/gendata.cwl#L7 and https://github.com/reanahub/reana-demo-root6-roofit/blob/master/workflow/cwl/fitdata.cwl#L7.

Reply:
Honestly, I don't know. It's true that we can, but without a validation that there is an environment defined somewhere in the workflow specification the user won't easily find out what was the error. This requires some planning IRL as I see it.

Reply:
I think it will not help at all since we completely ignore this field now (I just run cwl workflow without the environment field). Either way, if we keep it in `cwl` as you suggest, what is the reason to not keep it in `yadage`?

Reply:
Wait a moment, what will not help? I should have created this as an RFC, I left it on CWL because if you read the issue only yadage is reported to have a preset environment. If it is true for CWL as well then I think we should: 1. write issues that are complete 2. plan how to handle reana.yaml files in general on paper, as I was working without clear directions and the result is just more misunderstanding.

Reply:
I just thought that `yadage` and `cwl` reana.yaml files changes should be the same in this PR because in an abstract way they both follow the same pattern. Both are two formal specs which include the docker images inside (see [here for yadage](https://github.com/reanahub/reana-demo-root6-roofit/blob/master/workflow/yadage/workflow.yaml#L20) and [here for cwl](https://github.com/reanahub/reana-demo-root6-roofit/blob/master/workflow/cwl/fitdata.cwl#L7)) so we can run into the problem of missing docker image inside the spec in both cases, we can see IRL.